### DISCUSSION TITLE
Press "Enter" in "Command Mode" in version plugin 1.15 + 1.16 - dont working

### DISCUSSION THREAD

**Discussion by spouk** [amibiz/ergo-keys#41] [Q&A]
The question is is this a mistake or is it intended to be so, in ergokeys 1.14 - Goland 2023 version.2.5 everything works great. In version 1.15 + 1.16 - Goland 2023.3.2 pressing "Enter" in the "Command Mode" mode does not translate to a new line and does not work at all.

**Comment by amibiz** [comment]
Hey @spouk, thanks for noticing the change between versions. I will take a look at and get back to you.

As a workaround, you can bind `Enter` key in `CMD` mode to start a new line.

![image](https://github.com/amibiz/ergo-keys/assets/3448982/77ae43d5-309d-4db7-b79d-035a3e37f434)

**Comment by spouk** [comment]
Yes, that's exactly what was needed, thanks for the help!

**Comment by amibiz** [comment]
Reproduced with version 1.14 in PyCharm 2023.2.5

This is a regression, opened issue #42 to fix it.

Thanks @spouk
